### ISSUE TITLE
Trainz Railroad Simulator 2004: No video or no keyboard input

### ISSUE THREAD

**Issue by divadsn** [doitsujin/dxvk#2429]
The game doesn't show anything when running full screen, but the keyboard/mouse input and sound work, however when I run the game with the virtual desktop on, the game works fine, but the keyboard input doesn't work.

### Software information
Trainz Railroad Simulator 2004, obtained as abandonware from U7-Trainz.de

### System information
- GPU: AMD Radeon RX 5500 XT
- Driver: amdgpu (Kernel 5.15.12)
- Wine version: 7.0-rc2
- DXVK version: 1.9.2

### Apitrace file(s)
None at the moment, will supply later if required.

### Log files
- d3d9.log: [trainz_d3d9.log](https://github.com/doitsujin/dxvk/files/7836694/trainz_d3d9.log)
- d3d11.log:
- dxgi.log:

**Comment by doitsujin** [comment]
Does this work with wined3d? Keyboard input in particular is something we *should* not have any sort of control over.

**Comment by divadsn** [comment]
Yes it does work with wined3d fully, with fullscreen and on virtual desktop.

**Comment by doitsujin** [comment]
can we have an apitrace please?

**Comment by divadsn** [comment]
Yes of course, will send within 30 minutes

**Comment by divadsn** [comment]
Here we go, my upload is slow today: https://drive.google.com/file/d/1ur2zobHt_bAgh6WFV3HIKKwQQVpzwe0J/view?usp=sharing

Recorded this apitrace with fullscreen and no virtual desktop.

**Comment by divadsn** [comment]
The black fullscreen issue is also present on Windows, so that might be better to fix first.

So to sum up, without virtual desktop, the game launches with a black screen, but with sound and keyboard input working.

**Comment by divadsn** [comment]
Is there anything more required or missing to help fix this issue?

**Comment by Blisto91** [comment]
Can confirm the issue. 
Latest dxvk and master = black screen. Needs virtual desktop before anything displays.
Wined3d works fine without.
